.github/workflows/build.yaml

```diff
@@ -129,9 +129,9 @@ jobs:
                 export label_extension="-gaudi"
                 export docker_volume="/mnt/cache"
                 export docker_devices=""
-                export runs_on="ubuntu-latest"
+                export runs_on="itac-bm-emr-gaudi3-dell-8gaudi"
                 export platform=""
-                export extra_pytest=""
+                export extra_pytest="--gaudi"
```

Comment:
This will run the models with `"run_by_default": True` in PRs right?
If yes, I think we should change the runner above from `itac-bm-emr-gaudi3-dell-8gaudi` to `itac-bm-emr-gaudi3-dell-2gaudi` so that we test Llama 8b on a single device and on 2 devices.